qa/suites/rados/objectstore/backends/ceph_objectstore_tool.yaml

```diff
@@ -13,7 +13,7 @@ tasks:
         osd max object name len: 460
         osd max object namespace len: 64
       osd:
-        osd objectstore: filestore
+        osd objectstore: bluestore
         osd op queue: wpq
```

Comment:
I think entire file should be removed. 
And qa/suites/rados/objectstore/backends/filejournal.yaml too.

Reply:
removing filejournal.yaml
ceph_objectstore_tool.yaml - but there are still functions that left there that are not related to filestore, should we still consider removing it?

Reply:
@aclamk ?

Reply:
You are correct, lets leave it as is.

Reply:
@aclamk Thanks!